src/scss/components/accordion/_stepper.scss

```diff
@@ -25,44 +25,44 @@ hx-accordion.hxStepper {
       // TODO: clarify mouse hover/click behavior with Design
```

Comment:
Please update this block:

https://github.com/HelixDesignSystem/helix-ui/blob/28f00a163c8b114bac182408c21635867b7a133e/src/scss/components/accordion/_stepper.scss#L9-L14

Reply:
Changes have been done, please review.